netty/src/test/java/io/grpc/transport/netty/NettyClientTransportTest.java

```diff
@@ -79,7 +83,11 @@
  */
 @RunWith(JUnit4.class)
 public class NettyClientTransportTest {
+  private static final int MAX_STREAMS_PER_CONNECTION = 1;
```

Comment:
This doesn't seem like something we would want for all tests. Shouldn't just the one new test be configured with this?